.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+	// Use IntelliSense to learn about possible attributes.
+	// Hover to view descriptions of existing attributes.
+	// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+	"version": "0.2.0",
+	"configurations": [
+		{
```

Comment:
This is allows nodemon to restart inside the docker compose with the debugger attaching